packages/reports/src/Containers/reports.tsx

```diff
@@ -70,10 +94,7 @@ const Reports = ({
                 <PageOverlay header={localize('Reports')} onClickClose={onClickClose} is_from_app={is_from_derivgo}>
                     <DesktopWrapper>
                         <VerticalTab
-                            alignment='center'
```

Comment:
`alignment`, `id` and `classNameHeader` were deleted because `VerticalTab` component has no such props